.github/workflows/test-workflow.yml

```diff
@@ -2,6 +2,14 @@ on:
   push:
     branches:
       - integ-tests
+  workflow_call:
+    inputs:
+      call_local_action:
```

Comment:
This parameter is used because we can't use `if: ${{ github.event_name != 'workflow_call' }}`

https://github.com/actions/runner/issues/3146